google-cloud-functions/info.json

```diff
@@ -0,0 +1,397 @@
+          "type": "json",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "A binding binds one or more members, or principals, to a single role. Principals can be user accounts, service accounts, Google groups, and domains (such as G Suite). A role is a named list of permissions; each role can be an IAM predefined role or a user-created custom role.",
+          "value": [
+            {
+              "role": "roles/resourcemanager.organizationViewer",
+              "members": [
+                "user:eve@example.com"
+              ],
+              "condition": {
+                "title": "expirable access",
+                "description": "Does not grant access after Sep 2020",
+                "expression": "request.time < timestamp('2024-01-01T00:00:00.000Z')"
+              }
+            }
+          ]
+        },
+        {
+          "title": "Etag",
+          "name": "etag",
+          "description": "Specify the etag based on which you want to set the access control policy in Google Cloud Functions.",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Important: If you use IAM Conditions, you must include the etag field whenever you call setIamPolicy. If you omit this field, then IAM allows you to overwrite a version 3 policy with a version 1 policy, and all of the conditions in the version 3 policy are lost."
+        },
+        {
+          "title": "Policy Version",
+          "name": "version",
+          "description": "Specify the maximum policy version that will be used to format the policy. Valid values are 0, 1, and 3.",
+          "type": "integer",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Important: If you use IAM Conditions, you must include the etag field whenever you call setIamPolicy. If you omit this field, then IAM allows you to overwrite a version 3 policy with a version 1 policy, and all of the conditions in the version 3 policy are lost."
+        },
+        {
+          "title": "Update Mask",
+          "name": "updateMask",
+          "description": "Specify the comma-separated list of fully qualified names of fields.",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "A FieldMask specifying which fields of the policy to modify. Only the fields in the mask will be modified. If no mask is provided, the following default mask is used: paths: \"bindings, etag\".",
+          "placeholder": "e.g. user.displayName,photo"
+        }
+      ],
+      "output_schema": {}
```

Comment:
output_schema is missing for all actions

Reply:
We don't have setup access (paid services), this will release as uncertified.